biahub/tests/test_concatenate.py

```diff
@@ -387,7 +396,7 @@ def test_concatenate_with_mixed_slice_formats(create_custom_plate, tmp_path):
     )
 
 
-def test_concatenate_with_unique_positions(create_custom_plate, tmp_path):
+def test_concatenate_with_unique_positions(create_custom_plate, tmp_path, sbatch_file):
```

Comment:
Is this actually used?

Reply:
I wasn't sure if the argument in this test case gets used since there is only diff in the signature, not in the function body.

Reply:
Ah, yes, good catch, thanks. Fixed.